src/components/StyledText.tsx

```diff
@@ -12,9 +12,9 @@ const StyledText: FC<Props> = ({ children, error, style }) => {
     <p
       className={css(
         styles.text,
-        style,
         typography.body1,
-        error && styles.errorText
+        error && (styles.errorText, typography.error1),
+        style
```

Comment:
in this case should be : typography.body1, styles.input, styles, focus, error

Reply:
Ok